airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/932e6363-d006-4464-a9f5-102b82e07c06.json

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
why these getting deleted?

Reply:
These aren't present in `source_definitions.yaml`. I guess I should re-add them?

Reply:
done